addressing/README.md

```diff
@@ -147,6 +147,33 @@ within it. Using our example above, decapsulating either `/tcp/1234/ws` _or_
 unsurprising if you consider the utility of the `/ip4/7.7.7.7/ws` address that
 would result from simply removing the `tcp` component.
 
+### The security multiaddr
+
+Peers MAY advertise their addresses without the security protocol, e.g.
```

Comment:
```suggestion
Peers MAY advertise their addresses without a security protocol, e.g.
```

There are multiple security protocols, e.g. `tls` and `noise`, thus using `the` instead of `a` seems off here. I am not a native speaker, thus my intuition might be wrong.